File: code_templates/actions/php/UploadFile.php
move_uploaded_file($_FILES["fileData"]["tmp_name"], $_FILES["fileData"]["name"]);
$uploadToken = new KalturaUploadToken();
$result = $client->uploadToken->add($uploadToken);
$tok=$result->id;
$resume = null;
$finalChunk = null;
$resumeAt = null;
$result = $client->uploadToken->upload($tok, $_FILES["fileData"]["name"], $resume, $finalChunk, $resumeAt);
<%- Lucy.returnCode("$result") %>

File: code_templates/actions/ruby/UploadFile.rb
name = params[:fileData].original_filename
directory = "/tmp/"
path = File.join(directory, name)
File.open(path, "wb") { |f| f.write(params[:fileData].read) }
video_file=File.open(path)
uploadToken = KalturaUploadToken.new
result = @@client.upload_token_service.add(uploadToken)
token = result.id
result = @@client.upload_token_service.upload(token,video_file,nil,nil,nil)
<%- Lucy.returnCode('result') %>
